lib/phpseclib/bootstrap.php

```diff
@@ -13,10 +13,3 @@
         );
     }
 }
-
-spl_autoload_register(function($className) {
```

Comment:
this file should get removed by https://github.com/OpenMage/magento-lts/pull/2411